client/question/services/code_evaluators/PythonPrereqCheckService.js

```diff
@@ -107,7 +107,8 @@ tie.factory('PythonPrereqCheckService', [
       for (var i = 0; i < codeLines.length; i++) {
         var line = rightTrim(codeLines[i]);
         var VALID_LINE_PATTERN = /(^\s+)|(^def)|(^import)/;
-        if (line.search(VALID_LINE_PATTERN) === -1 && line !== '') {
+        if (line.search(VALID_LINE_PATTERN) === -1 && line !== '' &&
+            !line.startsWith('#')) {
```

Comment:
Think it's worth changing this additional check to "first non-space char on line is a '#'"?

Reply:
Sorry, ignore this -- thinko on my part. It's caught in VALID_LINE_PATTERN, and those comments would be in a different scope anyway.